pyiode/iode_database/global_funcs_database.pyx

```diff
@@ -8,6 +8,12 @@ from pyiode.iode_database.database cimport low_to_high as cpp_low_to_high
 from pyiode.iode_database.database cimport high_to_low as cpp_high_to_low
 
 
+# * use Enums for type_of_series and method instead of int and str (or use strings for both)
+# * use longer "constant" name for method: 'linear', 'cubic', 'steps'
+# * support Path objects for filepath
```

Comment:
This should be the case for all *filepath* in all methods and functions.
I will open an issue instead and do it later.
But it is a good advice.

Reply:
Indeed, that's what I meant 😉